mycroft/client/enclosure/__main__.py

```diff
@@ -70,8 +70,8 @@ def main(ready_hook=on_ready, error_hook=on_error, stopping_hook=on_stopping):
     only the GUI bus will be started.
     """
     # Read the system configuration
-    system_config = LocalConf(SYSTEM_CONFIG)
-    platform = system_config.get("enclosure", {}).get("platform")
+    config = Configuration.get(remote=False)
+    platform = config.get("enclosure", {}).get("platform")
```

Comment:
It's to deal with the many possible configs scenario yes. XDG comes with it's `/etc/xdg` system level configuration but Mycroft also supports plain `/etc`. Both of these have to be checked. We can just check those 2 directories manually here as well, which might make the `Configuration.get()` function cleaner.

Reply:
i think using all configs follows the [Principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)

If as a dev/user i change that string in the file i'm told belongs to user and it does nothing I will be scratching my head

Note that there are skills depending on this string, if you install some "enclosure skill" it will have write access to user config as part of its setup, but not to the system level.

This string is really a flag that changes behavior, its no longer a simple identifier about what hardware its running.  With the mark2 loading different QML and such based on this string i can also imagine skills wanting to override this to disable stuff like the UI scaling and providing a new "skin".

Reply:
I think I agree with Jarbas on this, if I am reading his comment right.  I have always been confused why certain parts of the system only need certain config files.  Why can't we just always use all the configs?  What benefit is there to only loading a partial list of the config files?

Reply:
Excluding remote is generally a good idea if we know that a setting will not originate from there. The speedup is significant.

Reply:
agree in excluding remote, especially because it should never touch these values, just disagree with the user config being removed

one is an optimization and you fully control it, the other is user facing and just confusing if ignored

Reply:
Agreed, I think this is good including the User config in this. I guess the old version was a quick hack. Is the consensus that we leave it like it is now, since as far as I can tell it includes all local config files?

Reply:
This includes all configs but the remote one yes. I agree with you guys here, exclude remote but otherwise use all available configs in the proper priority.

Reply:
One thing that still bugs me about this... a config that is not in the remote config right now could be added in the future.   For example, when the Mark II gets the the "consumer ready" point, we should have a remote config that allows users to select their language.  How do we know if doing so breaks this code (or any other code that excludes remote)?

One other question, why is parsing and extra JSON file and merging it with others such an expensive operation?